hotel/templates/emails/hotel_rooms.txt

```diff
@@ -6,7 +6,10 @@ Our records indicate that you've already filled out the hotel room form, so than
 {% else %}
 If you want hotel room space, you must sign up by {{ c.ROOM_DEADLINE|datetime }}.
 {% endif %}
-All volunteers who receive hotel room space must sign up for at least {{ c.HOTEL_REQ_HOURS }} weighted hours worth of shifts, but you won't be charged any money.  Let us know if you're interested in helping with setup and/or teardown shifts for {{ c.EVENT_NAME }}, and if we haven't already selected enough volunteers to help, you'll be approved for the extra night(s). Your department will have to approve these those nights and they can count towards the {{ c.HOTEL_REQ_HOURS }} hour requirement.
+All volunteers who receive hotel room space must sign up for at least {{ c.HOTEL_REQ_HOURS }} weighted hours worth of shifts, but you
+won't be charged any money.  Let us know if you're interested in helping with setup and/or teardown shifts for {{ c.EVENT_NAME }}, and
+if we haven't already selected enough volunteers to help, you'll be approved for the extra night(s). Your department head will have to
+approve these those nights and they can count towards the {{ c.HOTEL_REQ_HOURS }} hour requirement.
```

Comment:
You should probably remove the line breaks after "you" / "and" / "to", since you probably didn't mean for those to be there.